.gitlab-ci.yml

```diff
@@ -9,7 +9,7 @@ before_script:
   - apt-get -y update
   - apt-get -y install jq
   - pip install -r requirements.txt
-  - export DEPLOYMENT_ENV=$CI_COMMIT_REF_NAME
+  - export DEPLOYMENT_ENV=integration
```

Comment:
`$CI_COMMIT_REF_NAME` resolves to the deployment being tested in the scheduled dcp-wide integration tests, e.g. `integration`, `staging`, and `prod`. Shouldn't this change be reverted to continue testing in the other environments?

Reply:
Yes, this is a temporary change to just get the test running and passing in gitlab. It should be reverted once the changes are ready to merge

Reply:
An intermittent issue un-related to the changes caused the previous test failure for this branch. Will re-run and post a link to the completed pipeline